### ISSUE TITLE
Decide how we measure the success of the initiative

### ISSUE THREAD

**Issue by StevenMaude** [opensafely-core/codespaces-initiative#17]
We should specify some way of determining whether the use of Codespaces is an improvement.

**Comment by lucyb** [comment]
Dependent on #8

**Comment by lucyb** [comment]
New ticket created to measure Codespaces usage - #42.

Otherwise, I think we'll need to seek out feedback from copilots and researchers in a few months.
